package.json

```diff
@@ -5,7 +5,10 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "webpack"
+    "build": "webpack",
+    "watch-build": "webpack --watch --build",
+    "start-server": "node app.js",
+    "start-dev": "npm run watch-build && npm run start-server"
```

Comment:
Looks like && is for serial execution of the two scripts.  & is for parallel execution.

We want both scripts running at the same time, so change '&&' to '&'

Reply:
https://stackoverflow.com/questions/39172536/running-npm-scripts-sequentially